src/components/editor/MonacoEditor/index.tsx

```diff
@@ -222,23 +277,27 @@ function MonacoEditor({
         },
     };
 
-    if (catalogName && specAsString) {
+    if (catalogName && typeof defaultValue === 'string') {
         return (
             <Paper sx={{ width: '100%' }} variant="outlined">
                 <Box
                     sx={{
                         minHeight: toolbarHeight,
+                        py: 0.5,
+                        px: 1,
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        alignItems: 'center',
                     }}
                 >
+                    <Typography sx={{ fontWeight: 500 }}>
+                        {editorLabel}
+                    </Typography>
```

Comment:
If we're gonna label this we should eventually make this more like a fieldset or something for the editor. Not really sure the best way to make this accessible for Monaco.